pgcommitfest/commitfest/templates/base.html

```diff
@@ -25,12 +28,20 @@
     {%endif%}
     <li class="pull-right active">
      {%if user.is_authenticated%}
-      Logged in as {{user}} (<a href="/account/profile/">edit profile</a> | <a href="/account/logout/">log out</a>{%if user.is_staff%} | <a href="/admin/">administration</a>{%endif%})
+      Logged in as {{user}} (<a href="/account/profile/">edit profile</a> | <a href="/account/logout/">log out</a>{%if
+      user.is_staff%} | <a href="/admin/">administration</a>{%endif%})
      {%else%}
       <a href="/account/login/?next={{request.path}}">Log in</a>
      {%endif%}
     </li>
-    {%if header_activity%} <li class="pull-right active"><a href="{{header_activity_link}}">{{header_activity}}</a></li>{%endif%}
+    <li class="pull-right active">
+     <a href="https://github.com/postgres/pgcommitfest" target="_blank"
+        style="color: white; padding: 8px 12px; background-color: #24292e; border-radius: 5px; text-decoration: none;">
```

Comment:
This styling makes the link too obnoxious in my opinion. I think we should just use the normal styling, but probably add the github logo inline. You can add that using:
```html
<img class="github-logo" src="/media/commitfest/github-mark.svg"/>
```